src/Identity.Clients/Services/ClientService.cs

```diff
@@ -271,24 +276,29 @@ private void UpdateClaims(Data.Client entity, IEnumerable<ClientClaim> claims)
         {
             foreach (var claim in claims)
             {
+                claim.Type = claim.Type.Trim();
                 claim.Value = claim.Value.Trim();
 
                 if (claim.Id > 0)
                 {
                     var target = entity.Claims.SingleOrDefault(u => u.Id == claim.Id);
                     if (target != null)
                     {
-                        if (string.IsNullOrEmpty(claim.Value) || claim.Deleted)
+                        if (string.IsNullOrEmpty(claim.Type) || string.IsNullOrEmpty(claim.Value) || claim.Deleted)
```

Comment:
**Note:** currently this will erase all ClientClaims in the db that were made from the UI (if there are any) since they will have blank `Type` columns. 
If an issue, we should add a migration to update values of the db so Type and Value are used correctly there.